app/src/main/java/com/rootstrap/android/network/managers/ITargetPointManager.kt

```diff
@@ -0,0 +1,13 @@
+package com.rootstrap.android.network.managers
+
+import com.rootstrap.android.network.models.Target
+import com.rootstrap.android.network.models.TargetPointSerializer
+import com.rootstrap.android.network.models.TopicsSerializer
+import com.rootstrap.android.util.extensions.Data
+
+interface ITargetPointManager {
+
+    suspend fun createTarget(target: Target): Result<Data<TargetPointSerializer>>
+
+    suspend fun getTopics(): Result<Data<TopicsSerializer>>
+}
```

Comment:
Totally agree with you, you do not need this, 
managers are more to handle characteristics such as: 
Location, Storage, Database, Preferences, Notifications, etc ...
so this should be included in a ViewModel as you suggested.

Reply:
I renamed it to Target point service